modules/juce_audio_devices/native/juce_win32_WASAPI.cpp

```diff
@@ -1247,7 +1253,7 @@ class WASAPIAudioIODevice  : public AudioIODevice,
                 if (outputDevice == nullptr)
                 {
                     if (WaitForSingleObject (inputDevice->clientEvent, 1000) == WAIT_TIMEOUT)
-                        break;
+                        goto checkAudioSessionDisconnect;

```

Comment:
Yes, the default JUCE way of handling errors in audio I/O is quit the audio thread and not tell anyone.